charts/argo-rollouts/Chart.yaml

```diff
@@ -19,4 +19,4 @@ annotations:
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: fixed
-      description: Fixed rendering of plugins in the ConfigMap
+      description: Updated plugin values.yaml example and it's implementation to not need to include the stringification or the plugins block that it used to
```

Comment:
If you agree with https://github.com/argoproj/argo-helm/pull/3036/files#r1843061134 , `Correct type of metricProviderPlugins and trafficRouterPlugins as list` is clearer ?

Reply:
done